COMMERCE-10475 - Cannot paste text in the Quick Add to Cart search bar

victorcdp opened the pull request:
Related Issue: [COMMERCE-10475](https://issues.liferay.com/browse/COMMERCE-10475)

The issue was that the default behavior for the ClayMultiSelect element is to press Enter automatically when you paste something.
Proposed fix is to use the onPaste property to override this default behavior.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/1524/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1524/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:relevant - 27 out of 27 jobs passed in 1 hour 24 minutes 15 seconds 654 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8b6d68ffffb5ca46a0f58e5b75e2d4db6b94bb29">8b6d68ffffb5ca46a0f58e5b75e2d4db6b94bb29</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/42f0f00071090eef11549005abdeb1de66de4909">42f0f00071090eef11549005abdeb1de66de4909</a></p>27 out of 27 jobs PASSED<details><summary><strong>27 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/1524/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305543/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/318534/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305545/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/312124/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313904/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214538/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/65413/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289787/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/302387/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289786/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305544/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289785/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214537/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/278173/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/268347/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313903/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/302385/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/278174/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/268346/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305542/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/178574/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305546/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313902/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/268348/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/302388/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/302386/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1524/jenkins-report.html">here</a>.</h5></details></html>

brianikim commented:
Reviewing now :)

brianikim commented:
https://github.com/Alec-Shay/liferay-portal/pull/48

ChrisKian committed:
LPS-166987 LPS-167287 users-admin-impl: Fix UserIndexerTest.testLikeCharacter

Forward port of: LPS-140690 Solr throws errors if the booleanQueryImpl object uses an empty query, but other search engines expect this.  Searching for a blank string or '%' should return all results

https://issues.liferay.com/browse/LPS-167287

CalumR23 committed:
POSHI-245 Remove quotes on function

CalumR23 committed:
POSHI-245 Remove quotes from resource files

JanisZhang committed:
LPS-165626 Convert method reference to serviceTrackerList

Commit:
LPS-166987 LPS-167287 portal-search-test: Fix SearchEngineAdapterTest

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-web: Solr request/response is not JSON

https://issues.liferay.com/browse/LPS-167287

wcao20170619 committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Add target for SortFieldTranslator Reference

https://issues.liferay.com/browse/LPS-167287

Commit:
LPS-166987 LPS-167287 portal-search-solr8-impl: Fix testSearchQuoted

https://issues.liferay.com/browse/LPS-167287

Commit:
LPS-166987 LPS-167287 portal-search-test: IndexerClausesChangeTrackingTest, IndexerClausesComplexQueryPartTest and IndexerClausesPermissionTest are only meant to run with elasticsearch ("search.full.query.suppress.indexer.provided.clauses" is a Search Experineces feature). Assume.assumeTrue is being forward ported from 73x

https://issues.liferay.com/browse/LPS-167287

Commit:
LPS-166987 LPS-167287 portal-search-test: Make testGetSearchEngine work with Solr or Elasticsearch

https://issues.liferay.com/browse/LPS-167287

Commit:
LPS-166987 LPS-167287 portal-search-test: Is this right? Solr doesnt set SearchTimeValue

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-test: Remove ES7 dependency

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-test: Update test so it passes with Solr

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: I dont think this is necessary. Elastic should be blacklisted while Solr is active, so there should only be one reference

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-test: SF SearchEngineHelperImplTest

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Fix SearchResponseGetSearchTimeValueTest for Solr

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-test: This test should be able to pass with Solr

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: We should escape instead of unquote

https://issues.liferay.com/browse/LPS-167287

JanisZhang committed:
LPS-165626 Remove method getTransformerListener in JournalTransformerListenerRegistry as it only has 1 usage and apply.

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Prevent injection of Elastic translator

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-test: Fix assumes

https://issues.liferay.com/browse/LPS-167287

dsitu committed:
COMMERCE-10187 Add yaml changes for default payment method

dsitu committed:
COMMERCE-10187 Add rest methods

dsitu committed:
COMMERCE-10187 Add actions into responses

JanisZhang committed:
LPS-165626 Remove module journal-properties-transformer-listener, it is not deployed by default, also confirmed with echo team it is safe to remove

JanisZhang committed:
LPS-165626 Remove SF suppression

Commit:
LPS-169409 Change testray component for headless components that belong to Echo team

Commit:
LPS-169409 Add new Headless Echo component to existing routines

Commit:
LPS-169409 Add new echo-headless routine

Commit:
LPS-169409 Declare new echo-headless routine

Commit:
LPS-170051 - Add attachment and richtext to objectViewColumns.filter

Commit:
LPS-169409 SF

Commit:
LPS-169409 Add headless integration tests to echo batch

Commit:
LPS-169409 Add headless integration tests to echo-acceptance batch

Commit:
LPS-169409 Add headless integration tests to echo-headless batch

linolaoj committed:
LPS-171482 getting picklist options from object

dsitu committed:
COMMERCE-10187 Throw error when payment method is incorrect

aline-doleron committed:
LPS-171649 Create function to get the labels according to the locale

Co-authored-by: Igor França <igor.franca@liferay.com>

tinatian committed:
LPS-165626 SF, inline the only usage

joshuacords committed:
LPS-166987 LPS-167287 Fix faliures related to score

https://issues.liferay.com/browse/LPS-167287

joshuacords committed:
LPS-166987 LPS-167287 document-library-test: consider search engine during asserts

https://issues.liferay.com/browse/LPS-167287

tinatian committed:
LPS-165626 Remove unused method, the only usage was in journal-properties-transformer-listener which had been removed

CalumR23 committed:
POSHI-458 Update to Poshi Script

CalumR23 committed:
POSHI-458 Remove testcases

CalumR23 committed:
POSHI-458 Translate to Poshi Script

linolaoj committed:
LPS-171482 updating test

oliv-yu committed:
LPS-171715 search-experiences-web: Move default error constant to errorMessages

linolaoj committed:
LPS-171482 updating test

oliv-yu committed:
LPS-171715 search-experiences-web: Separate util functions

ealonso committed:
LPS-171788 Moves classes

ealonso committed:
LPS-171788 Unnecessary classes

ealonso committed:
LPS-171788 Removes no needed interface

ealonso committed:
LPS-171788 Moves logic to TrashHandler classes

ealonso committed:
LPS-171748 Removes no needed files

ealonso committed:
LPS-171748 Adds item selector for style books

ealonso committed:
LPS-171748 Uses item selector to select style books

ealonso committed:
LPS-171748 Adds item selector for master pages

ealonso committed:
LPS-171748 Uses item selector to select master pages

cgoncas committed:
LPS-160028 CheckIndividualSegmentsMessageListener should be immediate, otherwise the job won't be registered.

ealonso committed:
LPS-171748 Adapt logic to take into account the default entity

ealonso committed:
LPS-171748 Removes no needed custom styles

ealonso committed:
LPS-171748 It is no needed use ResourceBundleUtil since all of those keys are in portal-language-lang

ealonso committed:
LPS-171788 build.gradle

ealonso committed:
LPS-171788 baseline

oliv-yu committed:
LPS-171715 search-experiences-web: Update related tests for util functions

oliv-yu committed:
LPS-171715 search-experiences-web: Create types directory and remove outdated types

oliv-yu committed:
LPS-171715 search-experiences-web: Move language functions into a directory in utils

oliv-yu committed:
LPS-171715 search-experiences-web: Move validate functions into a directory in utils

oliv-yu committed:
LPS-171715 search-experiences-web: Move fetch functions into a directory in utils

Jeremy-Ch committed:
LPS-168610 Update macro to use variable

Jeremy-Ch committed:
LPS-168610 Add test for the nesting level of the kb navigation fragment

CalumR23 committed:
POSHI-458 Fix tests and macro

CalumR23 committed:
POSHI-458 Add new unit test

CalumR23 committed:
POSHI-461 Add check for testcase element

tinatian committed:
LPS-165626 SF, inline the only usage and delete unused classes

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Fix UserIndexerIndexedFieldsByAssociationTest.testAssociationsThatIndexMoreFields

https://issues.liferay.com/browse/LPS-167287

tinatian committed:
LPS-165626 Clean up on deactivate

tinatian committed:
LPS-165626 It should be volatile to ensure memory visibility

kenjiheigel committed:
POSHI-462 Update poshi file reading multithreading

kenjiheigel committed:
POSHI-462 Update poshi validating multithreading

kenjiheigel committed:
POSHI-462 Use ArrayList

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Fix UserIndexerIndexedFieldsTest

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 journal-test: Fix JournalArticleExpandoGeolocationSearchTest

https://issues.liferay.com/browse/LPS-167287

BryanEngler committed:
LPS-166987 LPS-167287 test.properties: Dont have to ignore this tests any more

https://issues.liferay.com/browse/LPS-167287

liulukiwi committed:
LPS-171748 Adapt macros for the new behaviors

Jeremy-Ch committed:
LPS-168639 Update macro for saving the reply message as draft

Jeremy-Ch committed:
LPS-168639 Update macro for viewing reply message status

Jeremy-Ch committed:
LPS-168639 Add test for the message board permission

jgarcialr committed:
LPS-171895 replace single D for dateFormat used by locale bg_BG

marcelabc committed:
LPS-171482 Condition is always 'false'

marcelabc committed:
LPS-171482 Declare the variable

lfbesada committed:
LPS-171754 Extract to a variable

lfbesada committed:
LPS-171754 Add language key

lfbesada committed:
LPS-171754 BuildLang

lfbesada committed:
LPS-171754 Add the success message

ealonso committed:
LPS-171805 Revert "LPS-171689 Adds add button only if we have one vocabulary"

This reverts commit 3fb9cf684846b50a07de0dbee9ae56c6528ceab8.

kevenleone committed:
LPS-171914 Improve getAll function with more options

welitonpe committed:
LPS-171915 Create Notification popover

kevenleone committed:
LPS-171914 Rename useHeader parameters

welitonpe committed:
LPS-171915 Sidebar enhancements

kevenleone committed:
LPS-171914 Upgrade SWR

WYZ-Issac committed:
LPS-168638 Add a test to checks that blog cover images in a staged environment will also display from repository in Live

haydenzheng314 committed:
LPS-171937 Remove usages of Optional<> in InfoItemHelper

haydenzheng314 committed:
LPS-171937 Apply usages

sandrodw3 committed:
LPS-171827 Don't prevent collection select when confirming

lfbesada committed:
LPS-169356 Add a condition to handle when the layout hasn't been published never

lfbesada committed:
LPS-169356 SF use bean reference instead of the util

lfbesada committed:
LPS-169356 Also take into account the limited update permissions

lfbesada committed:
LPS-169356 SF, cache in to a var and reuse

lfbesada committed:
LPS-169356 SF, the cheapest first

cgoncas committed:
LPS-171940 Update message, so the user is aware that one the scope is saved, it can't be changed.

lfbesada committed:
LPS-169356 Don't link to a page that will result in a 404 response because the layout hasn't any published version and either the user doesn't have update permission

ealonso committed:
LPS-157000 Calculate the modelPath instead of using the path, since in some situations in comes with a wrong groupId

ealonso committed:
LPS-157000 Removes unnecessary parameter

magjed4289 committed:
LRQA-78962 Add property portal.release

haydenzheng314 committed:
LPS-171937 SF

Commit:
LRQA-78956 Modify locators to adapt new markup

Commit:
LPS-171983 Clear sessionsStorage before opening a page in a new tab

Commit:
LPS-171761 resolve issue when the up or down buttons are clickable even if it was disabled.

Signed-off-by: Vy-CES <vy.bui@gs.liferay.com>

Commit:
LPS-171761 improve UX for the up and down buttons for the left box.

Signed-off-by: Vy-CES <vy.bui@gs.liferay.com>

Commit:
LPS-171984 Modify macro to make it reusable

JanisZhang committed:
LPS-172268 Remove Optional<> in LayoutListRetrieverContext

JanisZhang committed:
LPS-172268 Apply usages

JanisZhang committed:
LPS-172268 Remove Optional<> in StagedLayoutSet

JanisZhang committed:
LPS-172268 Apply usages

Commit:
LRQA-78954 Rename remote apps to client extensions

Commit:
LPS-169409 Rename Headless Echo testray component to Echo Headless

Commit:
LPS-169409 SF

victorg1991 committed:
LPS-171989 Avoid parsing the hash as an object

vendeltoreki committed:
LPS-163289 Semantic versioning

kevenleone committed:
LPS-171914 Update Types

kevenleone committed:
LPS-171914 Update Rest and useFetch with new API

kevenleone committed:
LPS-171914 Use REST and Fetch with new API

dsitu committed:
COMMERCE-10187 Use account exception from account entry

dsitu committed:
COMMERCE-10187 buildRest

welitonpe committed:
LPS-171914 Add Search Param conditinallly

welitonpe committed:
LPS-171914 Update Dispatch Trigger Status

welitonpe committed:
LPS-171915 SF

joshitagaki committed:
LRAC-12696 Test fix update script remove boolean argument

kenjiheigel committed:
POSHI-458 Rename and split up each test

kenjiheigel committed:
POSHI-458 Allow JSONCurlUtil to work with files

kenjiheigel committed:
POSHI-458 Fix unit tests

kenjiheigel committed:
POSHI-458 Use com.liferay.poshi.core:1.0.88

JanisZhang committed:
LPS-172268 Remove Optional usages in LayoutStructure

GaborKomaromi committed:
LPS-172119 using the portlet prefernce to get the delta instead of request

vendeltoreki committed:
LPS-163289 Do not skip layout set prototype merge when resetting merge fail count

vendeltoreki committed:
LPS-163289 Fix test cases by forcing layout set prototype merge

CalumR23 committed:
POSHI-245 Add in additional testcases

CalumR23 committed:
POSHI-245 Add check for content with multiple vars

BryanEngler committed:
LPS-166987 LPS-167287 portal-search-solr8-impl: Apply same logic as TermFilterTranslatorImpl so that queries can be parsed

https://issues.liferay.com/browse/LPS-167287

Jeremy-Ch committed:
LPS-170165 Add user to depot by JSON

Jeremy-Ch committed:
LPS-170165 Add role to user by JSON

Jeremy-Ch committed:
LPS-170165 Use fast navigation

Jeremy-Ch committed:
LPS-170165 Add user to site by JSON

ces-totrinh committed:
LPS-171281 Fix the contains function

Jeremy-Ch committed:
LPS-170165 Remove unnecessary steps

lfbesada committed:
LPS-171486 Adds the p_l_mode parameter to the previewURL

Jeremy-Ch committed:
LPS-170165 Rename the tests

lfbesada committed:
LPS-171486 SF remove unused

lfbesada committed:
LPS-171486 When the layout is type ControlPanel and p_l_mode is preview, look for the preview layout's plid as a portlet parameter

lfbesada committed:
LPS-171486 Makes the test faster, use the default company.

There is nothing in this test that justifies the need to create a specific company, all changes are made within the scope of the site and it is deleted after the test.

lfbesada committed:
LPS-171486 Adapt the assert method to allow testing with a different layout and add request parameters

lfbesada committed:
LPS-171486 Adapt the assert method to be able to verify that the client extension were not applied

lfbesada committed:
LPS-171486 Adds a test to assert the proper behavior when the layout is type ControlPanel and p_l_mode is preview

lfbesada committed:
LPS-171486 Enrich the ThemeDisplay to be able to get the static resource URLs

lfbesada committed:
LPS-171486 Adds required dependency

pei-jung committed:
LPS-171215 Make the order of the Filter by Status items consistent

dsitu committed:
COMMERCE-10187 Update status to NOT_FOUND

JanisZhang committed:
LPS-165626 baseline

JanisZhang committed:
LPS-165626 Fix integration test

JanisZhang committed:
LPS-172268 Remove Optional usages in LayoutActionsDisplayContext

JanisZhang committed:
LPS-172268 Simplify, getSegmentsExperienceId should never return null

JanisZhang committed:
LPS-172268 SF, segmentsExperience.getSegmentsExperienceId should never return -1

JanisZhang committed:
LPS-172268 Sem Ver

dsitu committed:
COMMERCE-10187 Fix patch test

gislayne-vitorino committed:
LPS-172285 (user-test) Set values with ReflectionTestUtil to prevent errors coming from Final variables

brookedalton committed:
LPS-172190 Set url flag when creating ES pass

dsitu committed:
COMMERCE-10187 Added integration test

CalumR23 committed:
POSHI-245 Update if to check if element is property

CalumR23 committed:
POSHI-245 Add additional testcases

CalumR23 committed:
POSHI-245 Delete quotes

kenjiheigel committed:
POSHI-245 Rework method and rename

kenjiheigel committed:
POSHI-245 SF

kenjiheigel committed:
POSHI-245 Fix tests

julschong committed:
LPS-172315 Remove Stream usages in JournalFolderLocalServiceImpl

IstvanD committed:
LPS-172281 Adding the available actions so that the management toolbar dropdown items can be checked and disabled accordingly

gamerson committed:
LPS-168898 fix bug in react app

gamerson committed:
LPS-168898 regen

melodywu committed:
LRQA-73479 Add macro User._editUserInformation and relpace

fmarcio committed:
LRAC-12616 Improve unit tests on Attributes and its Modal

fmarcio committed:
LRAC-12616 Improve unit tests on Connect and its Modal

fmarcio committed:
LRAC-12616 Improve unit tests on People

fmarcio committed:
LRAC-12616 Improve unit tests on Create Property Modal

fmarcio committed:
LRAC-12616 Create '__tests__' folder to Attributes and move files to it

fmarcio committed:
LRAC-12616 Delete Attributes tests TS files because they were moved to 'tests' folder

AliciaGarciaGarcia committed:
LPS-172117 decode the default canonical url

AliciaGarciaGarcia committed:
LPS-172117 decode uri from the url from the map of the canonical urls

fmarcio committed:
LRAC-12616 Create '__tests__' folder to Connect and move files to it

fmarcio committed:
LRAC-12616 Delete Connect and DisconnectModal tests TS files because they were moved to '__tests__' folder

fmarcio committed:
LRAC-12616 Create '__tests__' folder to People and move files to it

fmarcio committed:
LRAC-12616 Delete People and its Modals tests TS files because they were moved to '__tests__' folder

fmarcio committed:
LRAC-12616 Create '__tests__' folder to Properties and move files to it

fmarcio committed:
LRAC-12616 Delete Properties and its Modal tests TS files because they were moved to '__tests__' folder

pedro-oliveira446 committed:
LPS-172459 changing MDF Claim error message

fmarcio committed:
LRAC-12616 Create '__tests__' folder to Table and move file to it

fmarcio committed:
LRAC-12616 Delete Table test TS file because it was moved to '__tests__' folder

fmarcio committed:
LRAC-12616 Create '__tests__' folder to Components and move files to it

fmarcio committed:
LRAC-12616 Delete Components tests TS files because they were moved to '__tests__' folder

fmarcio committed:
LRAC-12616 Rename folder on DefaultPage tests

fmarcio committed:
LRAC-12616 Delete DefaultPage tests TS files because their folder was renamed

fmarcio committed:
LRAC-12616 Rename folder on utils tests

fmarcio committed:
LRAC-12616 Delete utils tests TS files because folder was renamed

fmarcio committed:
LRAC-12616 Create '__tests__' folder to js and move file to it

fmarcio committed:
LRAC-12616 Delete js test TS file because folder was renamed

fmarcio committed:
LRAC-12616 Remove types folder

fmarcio committed:
LRAC-12616 Regenerate types folder

fmarcio committed:
LRAC-12616 SF

joshitagaki committed:
LRAC-11262 Remove test from quarantine

joshitagaki committed:
LRAC-11263 Remove test from quarantine

david-truong committed:
LPS-172492 scheduled items can also be published in a publication

brianchandotcom committed:
LPS-166987 Sort

brianchandotcom committed:
LPS-166987 Wordsmith

brianchandotcom committed:
LPS-171940 Wordsmith

brianchandotcom committed:
LPS-171940 Regen

brianchandotcom committed:
LPS-171754 SF

brianchandotcom committed:
LPS-171754 Wordsmith

brianchandotcom committed:
LPS-171754 Regen

Commit:
LPS-172487 Fix Access Modifier

brianchandotcom committed:
LPS-171748 as used

brianchandotcom committed:
LPS-168898 auto SF

brianchandotcom committed:
POSHI-458 POSHI-461 Wordsmith

brianchandotcom committed:
POSHI-458 POSHI-461 prep next

brianchandotcom committed:
POSHI-458 POSHI-461 prep next

brianchandotcom committed:
LPS-171482 alphabetize

brianchandotcom committed:
POSHI-245 POSHI-462

Luizcarlosqueiroz committed:
LPS-172166 Change test assertion after fix

Luizcarlosqueiroz committed:
LPS-170217 Remove test from quarentine

Luizcarlosqueiroz committed:
LPS-171485 Remove test from ignored

Luizcarlosqueiroz committed:
LPS-170681 Remove test from quarantine

shuyangzhou committed:
LPS-172514 Remove unnecessary Language.properties in commerce-product-tax-category-web, added by ebec1fbbfbddf99f9d0237bdec0e5db8f9363aff , this "please-enter-a-unique-external-reference-code" key already exists in portal-language-lang

Commit:
LPS-172220 Add function to wait for text not present

Commit:
LPS-172220 Use new function

JanisZhang committed:
LPS-165626 Remove journal-properties-transformer-listener in test.properties

liulukiwi committed:
LPS-169806 Active staging from UI to stable the case

liulukiwi committed:
LPS-172266 Update case for the behavior change in LPS-165808

brianchandotcom committed:
LPS-165626 SF

brianchandotcom committed:
LPS-171486 Sort

brianchandotcom committed:
LPS-172281 SF

liferay-translation committed:
LPS-77699 Update Translations

Commit:
COMMERCE-10475 Add onPaste option to override default paste behavior in Commerce Mini Cart

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/045dfcd7cf0f5630008e30b11e4694570adea96a">045dfcd7cf0f5630008e30b11e4694570adea96a</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-10475">COMMERCE-10475</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/805d2f959e3b38af028bc5a01d4550cf93a6e494">805d2f959e3b38af028bc5a01d4550cf93a6e494</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/4476/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/4476/jenkins-report.html">here</a>.</h5><pre>     [exec] warning "workspace-aggregator-c58a38aa-ef26-4bcb-beca-2ade08b0b0ab &gt; frontend-taglib-clay &gt; @clayui/multi-step-nav@3.84.0" has unmet peer dependency "react-dom@^16.12.0".
     [exec] [4/4] Building fresh packages...
     [exec] Done in 7.97s.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.13.0
     [exec] \$ liferay-npm-scripts check
     [exec] Prettier checked 1 file
     [exec] 
     [exec] /opt/dev/projects/github/liferay-portal/modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartQuickAdd.js
     [exec]   156:28  error  'item' is not defined.  no-undef
     [exec] 
     [exec] ��� 1 problem (1 error, 0 warnings)
     [exec] 
     [exec] 1 of 3 jobs failed
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2023-01-11 18:01:19.260.
     [exec] 3 actionable tasks: 2 executed, 1 up-to-date
     [exec] 
     [exec] See the profiling report at: file:///opt/dev/projects/github/liferay-portal/build/reports/profile/profile-2023-01-11-10-00-57.html
     [exec] A fine-grained performance profile is available: use the --scan option.
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/4476/">test-portal-source-format#4476</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/4476/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/37">brianikim#37</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2157239595">[master] ci:test:sf - victorcdp &gt; brianikim - PR#37 - 2023-01-11[09:57:54]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/publish-testray-report/16600/">publish-testray-report#16600</a><br/></div></div>
